apps/web/src/components/dashboard/energy-consumption-card-chart.tsx

```diff
@@ -66,7 +79,7 @@ export default function EnergyConsumptionCardChart({ data, peaks, devices }: Pro
                 tooltip={{
                     content: EnergyConsumptionTooltip,
                 }}
-                xAxes={{ dataKey: "timestamp" }}
+                xAxes={{ dataKey: "timestamp", name: "Vergangene Zeit" + computeTimestampLabel(aggregation)}}
```

Comment:
Nur eine Kleinigkeit, aber besser wäre es, das Leerzeichen hier schon zu machen:
```suggestion
                xAxes={{ dataKey: "timestamp", name: "Vergangene Zeit " + computeTimestampLabel(aggregation)}}
```
Dann kannst du dir in Z. 52-57 jeweils das Leerzeichen zu Beginn des zurückgegebenen Strings sparen